test/EndToEndTests/Tests/Client/Microsoft.OData.Client.E2E.Tests/ActionOverloadingTests/Server/OrderLinesController.cs

```diff
@@ -0,0 +1,49 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="OrderLinesController.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved.
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.OData.Formatter;
+using Microsoft.AspNetCore.OData.Query;
+using Microsoft.AspNetCore.OData.Routing.Controllers;
+using Microsoft.OData.Client.E2E.Tests.Common.Server.EndToEnd;
+
+namespace Microsoft.OData.Client.E2E.Tests.ActionOverloadingTests.Server
+{
+    public class OrderLinesController : ODataController
+    {
+        public OrderLinesController()
+        {
+        }
+
+        [EnableQuery]
+        public IActionResult Get()
+        {
+            var products = CommonEndToEndDataSource.OrderLines;

```

Comment:
Why is this called products? Maybe you could show the actual type that's being returned instead of using `var` here to avoid confusion since it's not evident from the value.